dom.bs

```diff
@@ -4573,11 +4601,31 @@ dom-Range-extractContents, dom-Range-cloneContents -->
      <li><p><a lt="append an attribute">Append</a> <var>copyAttribute</var> to <var>copy</var>.
     </ol>
    </li>
+
+   <li>
+    <p>If <var>node</var> is a <a for=Element>shadow host</a> and its
+    <a for=Element>shadow root</a>'s <a for=ShadowRoot>clonable</a> is true:
+
+    <ol>
+     <li><p><a for=/>Assert</a>: <var>copy</var> is not a <a for=Element>shadow host</a>.
+
+     <li><p><a>Attach a shadow root</a> with <var>copy</var>, <var>node</var>'s
+     <a for=Element>shadow root</a>'s <a for=ShadowRoot>mode</a>, true, <var>node</var>'s
+     <a for=Element>shadow root</a>'s <a for=ShadowRoot>serializable</a>, <var>node</var>'s
+     <a for=Element>shadow root</a>'s <a for=ShadowRoot>delegates focus</a>, and <var>node</var>'s
+     <a for=Element>shadow root</a>'s <a for=ShadowRoot>slot assignment</a>.
+
+     <li><p>Set <var>copy</var>'s <a for=Element>shadow root</a>'s <a for=ShadowRoot>declarative</a>
+     to <var>node</var>'s <a for=Element>shadow root</a>'s <a for=ShadowRoot>declarative</a>.
+
+     <li><p><a>Clone a node's children</a> given <var>node</var>'s <a for=Element>shadow root</a>,
+     <var>copy</var>'s <a for=Element>shadow root</a>, <var>document</var>, and <var>subtree</var>.
```

Comment:
Don't we want shadow root cloning to happen after appending to avoid calling insertion steps again when the host is added to its parent?